docs/changelog.rst

```diff
@@ -51,6 +51,8 @@ Other changes:
 - Ensure the migration framework generates schema migrations for model fields that change the max_length
   (`gh-257 <https://github.com/django/django-localflavor/pull/257>`_). Users will need to generate migrations for any
   model fields they use with 'makemigrations'.
+- Lazily generate US_STATES, STATE_CHOICES, and USPS_CHOICES
+  (`gh-272 <https://github.com/django/django-localflavor/pull/272>`_).
```

Comment:
Could you also add a link to issue 203? You can see how to add 2 links in other changelog entries. Thanks.